core/src/main/java/nl/nn/adapterframework/pipes/IncreaseIntegerPipe.java

```diff
@@ -39,6 +45,7 @@
 
 	private String sessionKey=null;
 	private int increment=1;
+	private final static String INCREMENT = "increment";
```

Comment:
I would name this constant PARAMETER_INCREMENT, to indicate that it is not just the word 'increment', but meant as the name of a parameter